config/v1.3/calico.yaml

```diff
@@ -35,7 +37,7 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: quay.io/calico/node:v3.1.3
+          image: quay.io/calico/node:v3.3.5
```

Comment:
should be `quay.io/calico/node:v3.3.6` to be current on v3.3.x

Reply:
Bumped it, thanks.